bin/generateLocaleManifest.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+
+##
+# This script is designed to generate the chrome.manifest for the add-on in
+# the propert format, taking into account the locales present in the repo.
+##
+
+from __future__ import print_function
+
+import argparse
+import io
+import os
+import re
+import localeUtils
+
+# defaults
+DEF_L10N_SRC = os.path.join("locale")
+DEF_INPUT_FILE_NAME = os.path.join("locale", os.extsep.join(["jar", "mn"]))
+DEF_OUTPUT_FILE_NAME = os.path.join("built", "add-on", "chrome", "locale",
+                                    os.extsep.join(["chrome", "manifest"]))
+
+
+def main(l10n_src, input_file_name, output_file_name):
+    print("Using l10n tree from", l10n_src)
+
+    input_file = io.open(input_file_name, "r")
+
+    jar_mn = input_file.read()
+
+    # Drop any line that doesn't begin with %.
+    manifest_output = "\n".join(line for line in jar_mn.split("\n") if (re.match("^%", line)))
```

Comment:
Do we expect anything other than the one line of `% locale loop @AB_CD@ %locale/@AB_CD@/` ? If not, we could skip all the reading/replacing and just generate the manifest file with desired `locale` registrations.